src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomColor = new Random().nextInt(Colors.values().length);
+        return Colors.values()[randomColor].toString();
```

Comment:
Don't use `toString()` for getting `String` representation of enum constants. It's better to use standard method of enum `name()` that will be returning always `String` representation of concrete enum constant.